### DISCUSSION TITLE
Is it possible to increase the font of the menus?

### DISCUSSION THREAD

**Discussion by mongoose79** [dbeaver/dbeaver#21794] [Q&A]
![image](https://github.com/dbeaver/dbeaver/assets/21357104/57185e08-44aa-4683-9666-2acb7b5c8380)

**Comment by LonwoLonwo** [comment]
I think you can take a look at the Preferences -> User Interface -> Appearance -> Colors and Fonts

**Reply by mongoose79** [reply]
I didn't find which specific option should be modified:
![image](https://github.com/dbeaver/dbeaver/assets/21357104/fd9a76e7-dce1-4549-99e3-ec25039091d2)
